fluent-syntax/src/serializer.js

```diff
@@ -176,7 +184,13 @@ function serializePlaceable(placeable) {
     case 'Placeable':
       return `{${serializePlaceable(expr)}}`;
     case 'SelectExpression':
-      return `{${serializeSelectExpression(expr)}}`;
+      // Special-case select expression to control the whitespace around the
+      // opening and the closing brace.
+      return expr.expression
+        // A select expression with a selector.
+        ? `{ ${serializeSelectExpression(expr)}}`
+        // A variant list without a selector.
+        : `{${serializeSelectExpression(expr)}}`;
```

Comment:
should we use a single space before and after the expression?

Reply:
This is special-cased so that there aren't any spaces around it. Consider:

```properties
-brand-name =
    {    ← no space after the opening {
       *[nominative] Firefox
    }    ← no extra space before the closing {, just the 4-space indent
```